db/migrate/20230510132224_create_like.rb

```diff
@@ -1,5 +1,9 @@
 class CreateLike < ActiveRecord::Migration[7.0]
   def change
-    create_table :likes, &:timestamps
+    create_table :likes do |t|
+      t.references :posts, null: false, foreign_key: true
+
```

Comment:
- [ ] @djo1975 great work implementing the changes requested however you forgot to add a foreign key reference column for the user to make the association between the user and like complete.
![image](https://github.com/djo1975/Bloger/assets/61408860/377b7514-07f4-443e-a835-0c5639b8aff0)

Reply:
Thanks, @danmainah.